var/spack/repos/builtin/packages/totalview/package.py

```diff
@@ -19,67 +19,192 @@ class Totalview(Package):
     be used as the primary source and the architecture appropriate tarball will
     be downloaded as a resource."""
 
-    homepage = "https://www.roguewave.com/products-services/totalview"
-    maintainers = ["dshrader"]
+    homepage = "https://totalview.io"
+    maintainers = ["nicholas-sly", "dshrader", "petertea"]
     manual_download = True
     license_required = True
     license_comment = "#"
-    license_files = ["licenses/license.dat"]
-    license_vars = ["TV_LICENSE_FILE"]
+    license_files = ["licenses/license.dat", "licenses/tv_license_file"]
+    license_vars = ["LM_LICENSE_FILE", "TV_LICENSE_FILE"]
 
     # As the install of Totalview is via multiple tarballs, the base install
     # will be the documentation.  The architecture-specific tarballs are added
     # as resources dependent on the specific architecture used.
-    version("2021.1.16", sha256="4c51c7b6ab6b6afa7635ba2e9fc3b0ef833806f775a0ad0da26b13d6320625dd")
```

Comment:
I'll need to double-check on removing this... it is specifically used in TCE, but I don't know if TCE will keep its custom spack package for totalview.

Reply:
David,   It would be easy to put this back in place.   I was concerned on making this a very large package as TotalView releases 4 times a year and I thought I'd just keep the most recent ones in place.   But I have 2021.1.16, as well 2021.2.14 that could be added.  The original package had 2019.2.18 which was an interim build that is not publicly available.

Reply:
I've been trying to think about what would be the best way to manage this... the previous scheme was generic in the names of the archives to use, but the sha's definitely made each entry not generic. With so many archives per release, and each one needing a resource section, that is a lot to maintain. Getting rid of old versions may be problematic for some projects (e.g., TCE), but maybe those projects can manage it in some way.

Reply:
According to dshrader's comment
https://github.com/spack/spack/pull/33418#pullrequestreview-1189075145 all looks good now and he approved the PR. Setting this thread to resolved.